tests/ops/qutrit/test_qutrit_parametric_ops.py

```diff
@@ -0,0 +1,331 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for the available built-in parametric qutrit operations.
+"""
+from functools import reduce
+import pytest
+import copy
+import numpy as np
+from pennylane import numpy as npp
+
+import pennylane as qml
+from pennylane.wires import Wires
+from tests.ops.qubit.test_parametric_ops import NON_PARAMETRIZED_OPERATIONS
+
+from gate_data import TSHIFT, TCLOCK
+
+PARAMETRIZED_OPERATIONS = [
+    qml.TRX(0.123, wires=0, subspace=[1, 2]),
+    qml.QutritUnitary(TSHIFT, wires=0),
+    qml.ControlledQutritUnitary(TCLOCK, wires=[0], control_wires=[2]),
+]
+
+BROADCASTED_OPERATIONS = [
+    qml.TRX(np.array([0.142, -0.61, 2.3]), wires=0, subspace=[1, 2]),
```

Comment:
Seeing this made me wonder whether there are any situations in which a user would want to broadcast over both the rotation angles and the subspace, e.g.,
```
qml.TRX(np.array([0.142, -0.61]), wires=0, subspace=[[1, 2], [0, 1]])
```
I feel like that would not be common, since for broadcasting, the intention is that you're running multiple copies of a circuit with the _same_ structure, which to me implicitly includes the subspace. But what do you think?

Reply:
Thanks! Let's leave it then.